lib/hello/topic/comment.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Hello.Topic.Comment do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+
+  import Ecto.Query, warn: false
+  alias Hello.Repo
+  alias HelloWeb.Topic
+  alias Hello.Topic.Comment
+
+  schema "topic_comments" do
+    field :comment, :string
+    field :likes, :integer
+    field :reply_to, :integer
+    field :topic_id, :integer
```

Comment:
i think would use a 'belongs_to' association here, instead of just adding directly the topic_id field. Read more about it here https://hexdocs.pm/ecto/Ecto.Schema.html#belongs_to/3